### PR TITLE
CSI-1619 fix community csv documentation

### PR DESCRIPTION
**Pull request by oriyarde**
https://github.com/operator-framework/community-operators/pull/1632#issuecomment-621803330

https://operatorhub.io/preview

### CODE REVIEW

**Review by rivkap — APPROVED** [review]
@oriyarde  LGTM!

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -125,214 +125,7 @@ spec:
...
-    allowVolumeExpansion: true
-    ```
-
-    #### 3. Apply the storage class:
-
-    ```bash
-    $ kubectl apply -f demo-storageclass-gold-pvc.yaml
-    storageclass.storage.k8s.io/gold created
-    ```
+    For compatibility, prequities, release notes, and other user information, see the [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver).
```
Comment (rivkap) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml:128]: prerequisites (typo)

**Review by rivkap — APPROVED** [review]
approve with one typo fix needed

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml]: fixed

**Review by zingero — APPROVED** [review]

**Review by rivkap — APPROVED** [review]

**Pull request merged**